package.bzl

```diff
@@ -104,11 +104,31 @@ def rules_nodejs_dev_dependencies():
         sha256 = "d8b663c41039dfd84f3ad26d04f9df3122af090f73816b3ffb8c0df660e1fc74",
     )
 
+    http_archive(
```

Comment:
we don't want rules_nodejs to depend on rules_typescript as it makes it easier to get new layering violations

theres' a mock_ts_library in this repo for testing